### PR TITLE
Update team.md

### PR DESCRIPTION
**Pull request by khbsd**
Re-order team member item entries consistently, and move Koobs' Wiki link to his name header.

### CODE REVIEW

**Comment by koobs** [comment]
Always include a description :)

**Comment by khbsd** [comment]
Woops, got ahead of myself :) 10-4!

File: team.md [hunk 1]
```diff
@@ -5,39 +5,37 @@
...
-  - Experience: Telecom Field Tech, hobbyist computer enthusiast/programmer.
-  - Interests: 80s computers, UNIX, Programming, Infosec.
-  - Skills: Proficient in Python, Java, HTML and CSS. Can probably pick up whatever else is needed. Desktop Expert, customizer.
-  - Twitter: [@khbsd_](https://twitter.com/khbsd_)
-  - Timezone: CST, (GMT -0600)
+- About: Massive computer nerd. Loves old computers, UNIX, and BSD. Is 10% goofier than you.
+- Experience: Telecom Field Tech, hobbyist computer enthusiast/programmer.
+- Interests: 80s computers, UNIX, Programming, Infosec.
+- Skills: Proficient in Python, Java, HTML and CSS. Can probably pick up whatever else is needed. Desktop Expert, customizer.
+- Timezone: CST, (GMT -0600)
```
Comment (koobs) [inline team.md]: - Use consistent TZ format: `UTC[+-]XXXX` for all entries. Eg: My entry below is `UTC+XX`
- Probably worth putting country in, as timezone names alone remain a bit ambiguous (USA and AU has "EST" for example)

**Review by koobs — CHANGES_REQUESTED** [review]

File: team.md (hunk 1 above)
Comment (khbsd) [reply team.md]: good catch, just a sec

Comment (khbsd) [reply team.md]: Done!

**Comment by koobs** [comment]
Looks good!

**Comment by kfv** [comment]
I updated my entry, and it also requires some changes - my apologies. I also recommend moving @koobs's entry to sort them all alphabetically, if he permits.

**Pull request merged**